scene/2d/node_2d.cpp

```diff
@@ -474,6 +486,9 @@ void Node2D::_bind_methods() {
 	ADD_GROUP("Z Index", "");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "z_index", PROPERTY_HINT_RANGE, itos(RS::CANVAS_ITEM_Z_MIN) + "," + itos(RS::CANVAS_ITEM_Z_MAX) + ",1"), "set_z_index", "get_z_index");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "z_as_relative"), "set_z_as_relative", "is_z_relative");
+
+	ADD_GROUP("YSort", "y_sort_");
```

Comment:
We were discussed with some contributors that it would be good to group all sorting related properties together, so instead of the current "Z Index" and "YSort", you'd have a single "Sorting" group for all three properties.

Reply:
Done.